src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -1638,6 +1638,26 @@ public static Conversion ClassifyConversion(this SemanticModel? semanticModel, i
             var csModel = semanticModel as CSharpSemanticModel;
             return csModel?.GetInterceptorMethod(node, cancellationToken);
         }
+
+        /// <summary>
+        /// If 'node' cannot be intercepted syntactically, returns null.
```

Comment:
>syntactically [](http://example.com/codeflow?start=44&length=13)

What does it mean to be "intercepted syntactically"? Perhaps drop "syntactically" if it is not adding anything.

Reply:
`c.P()` where `P` is defined as `Action P { get; }` can be intercepted syntactically but an error will occur when you actually try to do it. Basically, the comment is meant to caution that even if you get a non-null location out, intercepting may still fail.